### PR TITLE
feat(popUp.tsx):Adding text in popup and fix of names in routes and p…

### PR DESCRIPTION
**Pull request by Fenerz07**
…ages

### CODE REVIEW

File: app/dons/page.tsx [hunk 1]
```diff
@@ -104,6 +104,9 @@ function DonsPage() {
                 onInputChange?: (
                     event: React.ChangeEvent<HTMLInputElement>,
                 ) => void
+                onInputChange?: (
```
Comment (Sterbenfr) [inline app/dons/page.tsx:107]: doublon de code

File: app/dons/page.tsx [hunk 2]
```diff
@@ -147,6 +150,12 @@ function DonsPage() {
                     value: null,
                     placeholder: 'Don de ',
                 }, //exemple: Dons de chocolat
+                {
```
Comment (Sterbenfr) [inline app/dons/page.tsx:153]: same

File: app/societe/[societeID]/groupe/page.tsx [hunk 3]
```diff
@@ -76,6 +77,9 @@ function GroupesPage({ params }: { params: { societeID: string } }) {
                             isPopUpOpen
                                 ? setIsPopUpOpen(false)
                                 : setIsPopUpOpen(true)
+                            isPopUpOpen
```
Comment (Sterbenfr) [inline app/societe/[societeID]/groupe/page.tsx:80]: same

File: components/popUp.tsx [hunk 4]
```diff
@@ -74,56 +74,207 @@ const PopUp: React.FC<PopUpProps> = ({ onClose, fields, url }) => {
...
-                                        handleInputChange(
-                                            input.id,
-                                            e.target.value,
-                                        )
-                                    }
-                                />
-                            )
-                    }
-                })}
+                <h2 className={style.lg}>Add New Entry</h2>
```
Comment (Sterbenfr) [inline components/popUp.tsx:243]: need to get back to the FR version

**Review by Sterbenfr — CHANGES_REQUESTED** [review]

**Review by MathisDacacio — APPROVED** [review]
LGTM

**Review by Sterbenfr — APPROVED** [review]
LGTM

**Pull request merged**